Library/ViewModels/PledgeViewModel.swift

```diff
@@ -111,3 +111,15 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
   public var inputs: PledgeViewModelInputs { return self }
   public var outputs: PledgeViewModelOutputs { return self }
 }
+
+private func toBase64(_ project: KsApi.Project) -> String {
```

Comment:
I don't think we actually need this. We have a protocol `GraphIDBridging` that defines:

```
extension GraphIDBridging {
  public var graphID: String {
    return Data("\(type(of: self).modelName)-\(self.id)".utf8).base64EncodedString()
  }
}
```

Looks like `Project` already conforms to it, you just need to add it to `Reward`.